python/az/aro/azext_aro/tests/latest/unit/test_validators.py

```diff
@@ -804,9 +817,15 @@ def test_validate_worker_vm_disk_size_gb(test_description, namespace, expected_e
     ),
     (
         "should not raise any Exception because namespace.client_secret is None and namespace.client_id is None",
-        Mock(client_secret=None, client_id=None),
+        Mock(upgradeable_to=None, client_secret=None, client_id=None),
         None
-    )
+    ), 
```

Comment:
```suggestion
    ),
```

Reply:
Thank you, Caden. Don't know why it wasn't caught on my machine.